lib/swm050/flash.c

```diff
@@ -0,0 +1,95 @@
+/** @addtogroup flash_file Flash peripheral API
+ * @ingroup peripheral_apis
+ * LGPL License Terms @ref lgpl_license
+ * @author @htmlonly &copy; @endhtmlonly 2019
+ * Caleb Szalacinski <contact@skiboy.net>
+ */
+/*
+ * This file is part of the libopencm3 project.
+ *
+ * Copyright (C) 2019 Caleb Szalacinski <contact@skiboy.net>
+ *
+ * This library is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ */
+/**@{*/
+#include <libopencm3/swm050/flash.h>
+
+#define IAP_WR	(void *)(0x1000AB)
+#define IAP_E	(void *)(0x100127)
```

Comment:
aren't these things a user might want exposed from flash.h too?

Reply:
I see what you're doing with them now, but they could be commented internally as poitners to ROM APIs?

Reply:
There isn't any documented use for these outside of these 3 functions.  Do you think they should be exposed?

Reply:
Fixed.